integrations/chroma/src/haystack_integrations/document_stores/chroma/document_store.py

```diff
@@ -248,9 +248,11 @@ def write_documents(self, documents: List[Document], policy: DuplicatePolicy = D
 
             if doc.content is None:
                 logger.warning(
-                    "ChromaDocumentStore can only store the text field of Documents: "
-                    "'array', 'dataframe' and 'blob' will be dropped."
+                    "ChromaDocumentStore cannot store documents without `content`. "
+                    "Document with id %s will be skipped.",
```

Comment:
Did something change about storing arrays and dataframes etc. Otherwise we should still include that in the warning?

Reply:
i improved the message now.

Reply:
nit: I would replace the "without" with something like: `ChromaDocumentStore cannot index empty Document(s) (i.e.: content=None)` but just a nit - all looks good to me as it